_studio/mfx_lib/shared/include/mfx_h264_enc_common_hw.h

```diff
@@ -942,6 +942,15 @@ namespace MfxHwH264Encode
 
     ENCODE_FRAME_SIZE_TOLERANCE ConvertLowDelayBRCMfx2Ddi(mfxU16 type, bool bTCBRC);
 
+    enum class SliceDividerType
+    {
+        OneSlice = 0,
+        Snb = 1,
```

Comment:
This enum doesn't make sense to me. No one, reading Hsw BluRay etc, will understand what it exactly means. I propose to do it this way: https://github.com/Intel-Media-SDK/MediaSDK/pull/1814/commits/91584f6455de9d008b3b02d5c354fc8fd1576871#diff-21db52ca9c332ae4833c901c5be556eaR995

Reply:
Done.